tests/writer/unit/core/test_writer_service.py

```diff
@@ -66,7 +66,7 @@ def test_writer_distribution(writer, database, occ_locker, read_database, messag
         "c/2/f": 4,
         "c/f": 5,
     }
-    write_request = WriteRequest(events, {}, 1, locked_fields)
+    write_request = WriteRequest(events, ["content"], 1, locked_fields)
```

Comment:
Isn't `information` a dict, mapping FQIDs to strings?

Reply:
We do not limit this field. The user can put anything in it: https://github.com/OpenSlides/openslides-datastore-service/blob/master/datastore/writer/flask_frontend/json_handlers.py#L41

Here is also just `JSON`: https://github.com/OpenSlides/OpenSlides/wiki/Datastore-Service